validator/src/main.rs

```diff
@@ -2145,6 +2145,8 @@ pub fn main() {
         expected_bank_hash: matches
             .value_of("expected_bank_hash")
             .map(|s| Hash::from_str(s).unwrap()),
+        // TODO: Once entrypoints are updated to return shred-version, obtain
+        // cluster shred version from ip-addr-server if this is None.
```

Comment:
Couldn't we plumb this code in this PR as well to avoid adding this TODO?  The current entrypoints would return `None` and `expected_shred_version` then remains None